scrapy/core/downloader/handlers/__init__.py

```diff
@@ -13,7 +13,7 @@ class DownloadHandlers(object):
     def __init__(self, crawler):
         self._handlers = {}
         self._notconfigured = {}
-        handlers = crawler.settings.get('DOWNLOAD_HANDLERS_BASE')
+        handlers = crawler.settings.get('DOWNLOAD_HANDLERS_BASE', {})
```

Comment:
We could use `crawler.settings.getdict()` (which has `{}` as default already) to get both 'DOWNLOAD_HANDLERS_BASE' and 'DOWNLOAD_HANDLERS'.